vertx-mongo-client/src/test/java/io/vertx/ext/mongo/MongoClientTestBase.java

```diff
@@ -75,6 +76,44 @@ public void testDropCollection() throws Exception {
   }
 
   @Test
+  public void testCreateIndex() {
```

Comment:
Hmmm...
`listIndexes` is the simplest way to verify that index has been created. I have no idea how to verify it differently